egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -213,6 +238,24 @@ def AddSigmoidLayer(config_lines, name, input, self_repair_scale = None):
     return {'descriptor':  '{0}_sigmoid'.format(name),
             'dimension': input['dimension']}
 
+def AddUnfoldedRnnLayer(config_lines, name, input, output_dim, ng_affine_options = "", self_repair_scale = None):
+    components = config_lines['components']
+    component_nodes = config_lines['component-nodes']
+
+    components.append("component name={0}_affine type=NaturalGradientAffineComponent input-dim={1} output-dim={2} {3}".format(name, input[0]['dimension'] + output_dim, output_dim, ng_affine_options))
+    self_repair_string = "self-repair-scale={0:.10f}".format(self_repair_scale) if self_repair_scale is not None else ''
+    components.append("component name={0}_tanh type=TanhComponent dim={1}".format(name, output_dim, self_repair_string))
```

Comment:
@vijayaditya Did you initialize the with identity matrix I, or (some small scale *I) as suggested in [1]?